server/apps/publish/formatters/aap_ipnews_formatter.py

```diff
@@ -175,115 +128,3 @@ def format(self, article, subscriber):
 
     def can_format(self, format_type, article):
         return format_type == 'AAP IPNEWS' and article['type'] in ['text', 'preformatted']
```

Comment:
Agreed that there are modules which have string literals. My comment was just related to this module only. 

I also agree that we need a backlog task for replacing these string literals globally with constant.